experimental/packages/opentelemetry-instrumentation-grpc/src/enums/AttributeValues.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+interface AttributeValuesType {
+  RPC_SYSTEM: string
+}
+
+export const AttributeValues: Readonly<AttributeValuesType> = {
+  RPC_SYSTEM: 'grpc'
```

Comment:
Same here

Reply:
Please check comments and what was before https://github.com/open-telemetry/opentelemetry-js/pull/3089#discussion_r925103565

and the same as I mentioned above